go/vt/hook/hook_test.go

```diff
@@ -2,9 +2,11 @@ package hook
 
 import (
 	"context"
+	"io"
```

Comment:
Also it seems like the license header is missing in this file, can you add it please?

Reply:
done